### ISSUE TITLE
Lowering the total block reward for SUB1X

### ISSUE THREAD

**Issue by SuB1X-Coin** [SuB1X-Coin/proposals#8]
### One time proposal
N/A

## Describe your proposal
I propose a significant reduction in the the total BR. During the bull market of 2017, the demand was there to sustain the price, despite the high levels of inflation in the coin.  Since Q1 of 2018,  demand for MN coins has dwindled and one of the largest contributing factors to this has been the high BR. 

There has been research conducted on the topic and Omni Data Analytics has come to the conclusion that coins with lower inflation rates have performed much better over the past 6 months, relative to those with higher ROI. 

https://twitter.com/OmniAnalytics/status/1041595763367849985

In my opinion, we should maintain the MN / PoS ratio and block target time, but opt for at least a 50% BR reduction. A vote can be held on determining whether there should be a drop and if the majority vote in favour, we hold another vote on the extent of the drop. 

I would really like to get some discussion from the community on this issue, as I feel it's something that must be addressed before release of the premium club.  Under planned budgetary allocations, it is already the case that over 90% of the governance budget will be in use for the next 6 months, adding 400 coins to the supply each month. 

### How is it going to help the project or the community?
It should mitigate the damage of MasterNode holders market selling on the exchange. Similar to mining, most MN holders choose to market sell to cover their VPS running costs, rather than place a bid and wait to get filled. This can be detrimental to the value of the coin over the long term. 

### Is this your first proposal? If not, please provide links to previous proposals.
Responsible for the marketing proposal and budgetary allocations for the premium club. 

## Proposal creator Discord user
willdono

**Comment by lucifer911** [comment]
I think it’s good idea. What would reward will reduce to?

**Comment by SuB1X-Coin** [comment]
No exact numbers, but I am thinking of a minimum reduction to 0.025 coins for the block.

**Comment by lucifer911** [comment]
That’s good. You should try to put cap on master nodes, so any given time there should not be more then few hundred . I think that would be good. Your thoughts?

**Comment by thewatchers1456** [comment]
I do like the idea of reducing the block reward as I was never in favor of the increase back in February, but will this reduction in block reward have an impact on the governance budgeting?

My understanding is that the governance budget will not exceed 20% of the monthly (43.2k block) block reward. With the current inflation rate there is a budget of a little over 400 coins per month.

My concern is that any reduction of this in the short term will greatly reduce the number of influencers SUB1X will be able to accommodate. 

I do agree that the BR should be reduced but we should take the next few months to discuss the best rate to shift it to and then shortly before the portal launch institute the change.

**Comment by SuB1X-Coin** [comment]
The superblock rewards can also be adjusted. The 20% is not set in stone and if a block reward issuance reduction were to take place, then we would raise the relative size of the SBR to ensure that the planned budgetary allocations from our WP are met.

I am not in favour of making these changes right before portal release. We need the coin, community and network to be stable in the last 2 months building up to the portal. I suggest that we get this out the way sooner, rather than later. The issuance reduction would activate at a certain block, most likely around 30 days after the 2nd proposal to ensure that the majority of users are upgraded to the latest protocol, otherwise we could risk forking the chain and creating all sorts of complications. 

####That would give the following timeline:
If this current proposal is pushed through for voting today or tomorrow, there will still be well over 2 weeks for the community to vote with their nodes. 

The vote would conclude at the start of next month and we can then open up another GitHub issue to determine the new rewards structure. Assuming the community push the 2nd proposal through, we will then have to deliberate on the block of activation, but as stated previously, it will be at least 1 month after the conclusion of 2nd round voting. 

At best we'd be looking at Jan 2019 for the new rewards and this would only be several months, before portal release.

**Comment by ghost** [comment]
I'm glad that the MN owers will have the opportunity to vote on such important decisions for the network. The new block size can be voted as well. Currently `0.05` SUB1X per block. I recommend posting 3 proposals:
* `block-reward-0.05` this is to vote on keeping the current block reward
* `block-reward-0.025`
* `block-reward-0.01`
 
All three proposals can use the same proposals URL

**Comment by ghost** [comment]
We could also have a vote on the MN collateral, for example:
* `mn-collateral-20`  << to maintain the MN collateral at 20 SUB1X
* `mn-collateral-50`
* `mn-collateral-100`

this is another way to reduce the MNs in the network. Make them something special, only for those that are willing to be part of the project and help out. Againt it will be voted democratically.

**Comment by lucifer911** [comment]
Do not change the Collateral at all. Keep it to 20.

**Comment by ghost** [comment]
@lucifer911 that's your opinion.
I can say as well that reducing the block rewards is a bad idea.

But why not ask the MN owners this question via a proposal? Afraid of the answer?

**Comment by lucifer911** [comment]
Yes you are right it is my opinion as a Master-node holder, and no NOT Afraid fo any answers, Any one can voice there opinion.

**Comment by ghost** [comment]
Good, what MN collateral values should we consider for voting?

**Comment by lucifer911** [comment]
If It's me I will try following tier system.

- `mn-collateral 20 Zsubx1 = block-reward 0.01 Zsubx1`

- `mn-collateral 30 Zsubx1 = block-reward 0.02 Zsubx1`

- `mn-collateral 40 Zsubx1 = block-reward 0.03 Zsubx1`

Again, It just me saying it. If it make any sense, then good, if not, then forget it. :)

**Comment by pioniere** [comment]
I am in favor of moving forward with the Block Reward proposal. The MN collateral proposal is a separate issue and is not relevant to this discussion.

**Comment by ghost** [comment]
Agree, MN collateral increase should be a different proposal. Will create a ticket for it

**Comment by Thatdudesbro** [comment]
I am not currently a MN operator, however I do feel strongly that a reduction in BR would be beneficial. After all, what would be the point of being the "lowest initial supply MN coin" if it inflates this fast? Why buy the coin from an exchange, if you can just mint it yourself?

I also believe the collateral should be raised and a limit on active MNs should be implemented. Lets say there can be a max of 255 MN and the min you could lock in as collateral is 20 coins; a MN brought online with more collateral could then have a higher priority and lesser valued nodes would be taken down.
This could discourage the selling off of BRs.

**Comment by SuB1X-Coin** [comment]
@Thatdudesbro We cannot limit the amount of MasterNode holders because that would limit the growth potential of the premium club.  One of the reasons that we proposed portal creation in the first place was as a means for making the nodes scarce, and therefore more valuable. We can't do that if there is a fixed limit in place.

**Comment by selerum** [comment]
instedt of reducing the block reward;
i would still highly suggest and vote for a dramendustly increasement of the mn collateral.

**Comment by aaroque** [comment]
IMHO as a MN holder and involved in this project since January this reduction will be very important for the future and beyond. 
Saw this option : 

>> block-reward-0.01   <<

Love it. Is a very good significant reduction and will give Sub1x extra scarcity as well as place our 1 million coin supply way beyond our 2056 estimated time. 

Do we have a time line for this.?? As SuB1X-Coin said the sooner the better, after the conclusions will be at least 1 month to have everything ready for testing and implementation, isn't..?
Same opinion regarding the MN collateral proposal, is a separate issue, not relevant to this discussion and once we see the results with the new Block Rewards we can decide about it but I'm 99.5% sure that with that BR reduction we won't need to touch the MN collateral, 20 coins collateral is affordable ATM for anyone interested and willing to be part of the project that can visualize the future ahead of it. 

That been said I think the proposal should be pushed through. Let our MN holders and Community members decide which path we should take.

**Comment by Thatdudesbro** [comment]
After speaking with @TheWatchers1456 I realize limiting the # of MNs would also limit # of Premier Portal users.

**Comment by SuB1X-Coin** [comment]
There has been some meaningful discussion on this issue. I am going to close it in a couple of days and submit the proposal on the 1st of November, giving node holders a full month to vote.

**Comment by lucifer911** [comment]
Done My Vote.

**Comment by Fibroblasto** [comment]
Note that with the reduction of the block reward the masternodes will be less profitable, at least in the short term (unless, of course, that the price rise and rise). People has to pay the VPS and might decide not buying coins for a not so profitable masternode. That's why I think it is very important to increase the collateral of the masternodes, as the smaller number of masternodes will make them more profitable and will invite people to buy coins for a new masternode.
I would increase the collateral at least three times.

**Comment by Klaud07** [comment]
The idea of a large reward was immediately questionable, so I support reducing the reward. It is also necessary to increase the collateral input. The price for a node at the start of the project was ~ $ 1700, so the increase in collateral should not prevent people from starting nodes, even if the collateral input is increased ( now the price of the node is only ~ $70). Perhaps even the number of nodes will not decrease much, because at the moment people just do not want to rent a lot of servers and they support POS. But here it is necessary not to forget about safety chains, although at the moment I don't see the problem. The ideal security solution would be a multi-level node system: a node: 20-40 coins, a supernode of 100-200 coins. But I'm not sure how realistic it is to implement...

**Comment by lucifer911** [comment]
Supernode is good idea.
